rewrite-java/src/main/java/org/openrewrite/java/AnnotationMatcher.java

```diff
@@ -74,7 +74,7 @@ private boolean matchesNamedParameters(J.Annotation annotation) {
             return true;
         }
 
-        if (annotation.getArguments() == null) {
+        if (annotation.getArguments().isEmpty()) {
```

Comment:
These can be null e.g. `@Deprecated` where empty when `@Deprecated()`

Reply:
Nullability changes undone